pkg/registry/core/node/storage/storage_test.go

```diff
@@ -165,125 +192,58 @@ func TestShortNames(t *testing.T) {
 }
 
 func TestResourceLocation(t *testing.T) {
-	testCases := []struct {
+	type TestCase struct {
```

Comment:
same here, don't export them if not necessary 😄

Reply:
my bad, missed that one i guess, fixed.